README.md

```diff
@@ -36,7 +38,9 @@ To run PlaceCal locally you will need:
 
 ## Quickstart with docker for GFSC devs
 
-Make sure all of the above dependencies are installed and ask someone to add your public ssh key to the server.
+First you need to append `lvh.me *.lvh.me` to your `/etc/hosts` file as a DNS alias for `127.0.0.1`.
```

Comment:
This isn't necessary - I haven't done this on my machine and everything works fine running from lvh.me, so I think this should come out.

The clarification below this is also not needed as this section is only for gfsc devs (I titled it 'quickstart with docker for gfsc devs') & without making some changes to the make file to add a non-staff command as well I don't think it will work for anyone else. If you think the title is confusing maybe you could just change that from 'devs' to 'staff' instead?

Reply:
i put these back how they were.